spec/acceptance/ini_setting_spec.rb

```diff
@@ -23,20 +23,17 @@
 
     describe file(path) do
       it { should be_file }
-      #XXX Solaris 10 doesn't support multi-line grep
-      it("should contain #{content}", :unless => fact('osfamily') == 'Solaris') {
-        should contain(content)
-      }
+      its(:content) { should match content }
```

Comment:
should contain still uses the specinfra matcher, however the serverspec its(:content) would use the https://github.com/serverspec/serverspec/blob/5b7e90f259305726527253744f7a0feb3de1cae4/lib/serverspec/type/file.rb#L97-L102 and read as a string and then use rspec match regex to match the multiline string.